### PR TITLE
Max-width and production match bugfix

### PR DESCRIPTION
**Pull request by dartharaknus**

### CODE REVIEW

**Comment by dartharaknus** [comment]
After this is done, we need to fix pagination and safari issues, and I think we can move forward with release.

File: src/components/Wordcloud/Wordcloud.jsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ Wordcloud.defaultProps = {
 };
 
 const mapStateToProps = state => ({
-  width: state.browser.width <= 960 ? state.browser.width - 50 : state.browser.width - 100,
+  width: state.browser.width <= 960 ? state.browser.width - 50 : (state.browser.width > 1540 ? 1440 : state.browser.width - 100),
```
Comment (howardchung) [inline src/components/Wordcloud/Wordcloud.jsx]: Math.min()?

**Comment by dartharaknus** [comment]
Updated.

File: src/components/App/App.css [hunk 2]
```diff
@@ -19,9 +19,15 @@
 .body {
   padding: 50px;
   flex-grow: 1;
+
   @media only screen and (max-width: 960px) {
     padding: 50px 25px;
   }
+
+  @media only screen and (min-width: 1540px) {
```
Comment (howardchung) [inline src/components/App/App.css]: Does this work for homepage (should not apply)?

Comment (dartharaknus) [inline src/components/App/App.css]: It is for homepage too and yes, homepage looks better like this

**Pull request merged**